api-admin/src/main/java/bio/terra/pearl/api/admin/service/portal/PortalExtService.java

```diff
@@ -60,28 +61,42 @@ public PortalEnvironmentConfig updateConfig(
       throw new PermissionDeniedException(
           "You do not have permissions to update portal configurations");
     }
-    Portal portal = authUtilService.authUserToPortal(user, portalShortcode);
+    authUtilService.authUserToPortal(user, portalShortcode);
     PortalEnvironment portalEnv = portalEnvironmentService.findOne(portalShortcode, envName).get();
     PortalEnvironmentConfig config =
         portalEnvironmentConfigService.find(portalEnv.getPortalEnvironmentConfigId()).get();
+    if (!user.isSuperuser()) {
+      if (!Objects.equals(config.getParticipantHostname(), newConfig.getParticipantHostname())
+          || !Objects.equals(config.getEmailSourceAddress(), newConfig.getEmailSourceAddress())) {
+        throw new PermissionDeniedException(
+            "ParticipantHostname and emailSourceAddress can only be updated by superusers");
+      }
+    }
```

Comment:
This stuff seems unrelated to pre-reg, but I assume it's something you want in the upcoming release?

Reply:
ah yup, this belongs in a different PR -- removing